external/sql/storm-sql-core/src/test/org/apache/storm/sql/compiler/backends/trident/TestPlanCompiler.java

```diff
@@ -274,4 +276,9 @@ private void waitForCompletion(long timeout, Callable<Boolean> cond) throws Exce
       Thread.sleep(100);
     }
   }
+
+  private void assertArrayWithoutOrder(List<Values> expected, List<List<Object>> actual) {
+    Assert.assertTrue("Two arrays are not same!\n"+ "Expected: " + expected + "\n" + "Actual: " + actual,
+            expected.containsAll(actual) && actual.containsAll(expected));
```

Comment:
@satishd Yes I missed the duplicated items. 
Btw, if we assume the duplication, checking size equality seems not enough too, since `[a b b a a]` and `[b b a a b]` is same length but it's not actually same. 
Checking cardinality of each element seems to be the safest way, and if the type of element is comparable, we can copy & sort two arrays and compare with ordered way.